packages/walletkit/src/new-web3-utils.ts

```diff
@@ -0,0 +1,144 @@
+import {
+  Callback,
+  ErrorCallback,
+  JSONRPCRequestPayload,
+  Subprovider,
+  Web3ProviderEngine,
+} from '@0x/subproviders'
+import debugFactory from 'debug'
+import Web3 from 'web3'
+import { JsonRPCResponse, Provider } from 'web3/providers'
+import { CeloProvider } from './transaction-utils'
+
+const debug = debugFactory('kit:web3:utils')
+
+export function generateAccountAddressFromPrivateKey(privateKey: string): string {
+  if (!privateKey.toLowerCase().startsWith('0x')) {
```

Comment:
We have a util for this: 
https://github.com/celo-org/celo-monorepo/blob/master/packages/utils/src/signatureUtils.ts#L90

Reply:
Thanks. That one is broken since it is not checking for `0X`. I will fix it in this PR and use it.